src/fb.js

```diff
@@ -53,24 +53,34 @@ var {version} = require('../package.json'),
 		return JSON.parse(header['x-page-usage']);
 	},
 	buildRateLimitObjectFromJson = function(rateLimit) {
-		return Object.assign(Object.create(null), {
```

Comment:
All this change does is remove the part of the code that stops this from inheriting from `Object.prototype`.